addons/web/static/src/legacy/js/views/kanban/kanban_record.js

```diff
@@ -708,6 +703,11 @@ var KanbanRecord = Widget.extend(WidgetAdapterMixin, {
             }
             elem = elem.parentElement;
         }
+        if (this.$el.hasClass('o_currently_dragged')) {
```

Comment:
I'd rather do that at the beginning of the function, and just `return`
Also, correct me if I'm wrong, but it's not a FIX, but a REF to ease extension (s.t. you don't need to add the protection if you override _openRecord, like it's done in project). I don't see how it's related to using Firefox though.

Reply:
Well for me this is both a FIX and a REF, the first change of this PR was for sure only a FIX but the second changes is a small REF that solves a bug (hence also a FIX). If you want me to change the tag to REF no problem but then I will probably also change the description and title so that it's not specific to the related project bug. For the Firefox I don't know why but the original bug only happens on Firefox.

Reply:
Agree